sqlcapture/discovery.go

```diff
@@ -75,45 +75,43 @@ func DiscoverCatalog(ctx context.Context, db Database) (*airbyte.Catalog, error)
 					Required: table.PrimaryKey,
 				},
 			},
-			Type: &jsonschema.Type{
-				AllOf: []*jsonschema.Type{
-					{
-						Extras: map[string]interface{}{
-							"properties": map[string]*jsonschema.Type{
-								"_meta": {
-									Type: "object",
-									Extras: map[string]interface{}{
-										"properties": map[string]*jsonschema.Type{
-											"op": {
-												Enum:        []interface{}{"c", "d", "u"},
-												Description: "Change operation type: 'c' Create/Insert, 'u' Update, 'd' Delete.",
-											},
-											"source": sourceSchema,
-											"before": {
-												Ref:         "#" + anchor,
-												Description: "Record state immediately before this change was applied.",
-												Extras: map[string]interface{}{
-													"reduce": map[string]interface{}{
-														"strategy": "firstWriteWins",
-													},
+			AllOf: []*jsonschema.Schema{
+				{
+					Extras: map[string]interface{}{
+						"properties": map[string]*jsonschema.Schema{
+							"_meta": {
+								Type: "object",
+								Extras: map[string]interface{}{
```

Comment:
It seems a little weird to use `Extras` for these instead of `Properties`. Is there some reason why `Properties` wouldn't work? Probably worth adding a comment, assuming this was intentional.

Reply:
@psFried there is a comment above that explains why